pages/about/groups/task-forces/pdf-aam.md

```diff
@@ -0,0 +1,92 @@
+
+### Work Statement
+
+The [PDF-AAM Task Force Work Statement](/about/groups/task-forces/pdf-aam/work-statement/) defines the initial objective, scope, approach, and participation of the Task Force.
+
+## Meetings and Communication
+
+The PDF-AAM Task Force conducts its work using a variety of synchronous and asynchronous tools.
+
+- Teleconferences of the Working Group and its task forces (also see [meeting minutes](https://www.w3.org/WAI/ARIA/task-forces/pdf-aam/minutes));
+- Face to face meetings;
+- [Email lists](#mailing-lists);
+- IRC discussion on the [#pdf-aam](irc://irc.w3.org/pdf-aam) IRC channel, used largely for minute-taking;
+- [PDF-AAM source repository issue tracker](https://github.com/w3c/pdf-aam/issues).
+- [ARIA specifications repository](https://github.com/w3c/aria);
+- [Web-Based Surveys (WBS)](https://www.w3.org/2002/09/wbs/159734/);
+
+These tools are used by participants of the Task Force. For ways non-participants can contribute, see [how to contribute to the Working Group and file comments](/about/groups/ariawg/contribute/).
+
+### Teleconferences
+
+The PDF-AAM Task Force meets the third Wednesday of the month, 1:00 - 1:30 PM Eastern Time. For more details, see [PDF-AAM Calendar](https://www.w3.org/groups/tf/pdf-aam/calendar).
+
+### Meeting Minutes
+
+[Minutes from previous meetings](https://www.w3.org/WAI/ARIA/task-forces/pdf-aam/minutes) are available.
+
+### Mailing Lists
+
+The PDF-AAM Task Force uses the public-pdf-aam@w3.org mailing list ([mailing list archives](http://lists.w3.org/Archives/Public/public-pdf-aam/)) for email discussion. Participants are automatically added to the mailing list when they become a participant of the Task Force.
+
+## Current Work
+
+The [Editors Draft](https://w3c.github.io/pdf-aam/) is expected to contain current PDF Accessibility API mappings.
+
+## Publications
+
+The PDF-AAM Task Force is primarily developing [PDF-AAM](https://w3c.github.io/pdf-aam/). See also the [PDF-AAM Task Force GitHub repository](https://github.com/w3c/pdf-aam/).
+
+## How to Comment, Contribute, and Participate
+
+To join the PDF-AAM Task Force, individuals must be participants of the [ARIA WG](/about/groups/apawg/). Participants are expected to [actively contribute](/about/groups/task-forces/pdf-aam/work-statement/#participation) to the work of the Task Force. If you are interested in participating in the PDF-AAM Task Force, please send e-mail to: [Duff Johnson, Valerie Young](mailto:duff.johnson@pdfa.org,spectranaut@igalia.com?subject=PDF-AAM%20Task%20Force%20Enquiry) and include a little bit about what you’re interested in and how you think that you may be able to contribute to the Task Force. Then follow the [ARIA Working Group participation](/about/groups/ariawg/participate/) procedures to join the Working Group, and once you have joined ask [Daniel Montalvo](mailto:dmontalvo@w3.org) to add you to the task force.
```

Comment:
re: ... If you are interested in participating in the PDF-AAM Task Force, please send e-mail to: [Duff Johnson, Valerie Young](mailto:duff.johnson@pdfa.org,spectranaut@igalia.com?subject=PDF-AAM%20Task%20Force%20Enquiry) ...

I think we want all these CCed to a list so we are aware of them.

/me debates which list... suggests wai@w3.org

Happy to discuss as useful.